app/models/concerns/moneyable.rb

```diff
@@ -0,0 +1,19 @@
+module Moneyable
```

Comment:
This is my first time coming acress the `money-rails` gem. It's a much nicer solution.